lightning/src/ln/channelmanager.rs

```diff
@@ -4594,6 +4610,10 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 				if let Some(msg) = channel_ready {
 					send_channel_ready!(channel_state.short_to_chan_info, channel_state.pending_msg_events, chan, msg);
 				}
+				if emit_channel_ready_event {
```

Comment:
Not sure I can follow entirely. IIUC, this would be an issue if a channel monitor update would be pending/in progress? But the bool would only ever be emitted when this would not be the case, as of
https://github.com/tnull/rust-lightning/blob/25269755c9a7ac82ef101f1b76731bf61bddb433/lightning/src/ln/channel.rs#L4937-L4946

But probably I'm misunderstanding something?

Reply:
Alright, now not bothering in funding methods as of https://github.com/lightningdevkit/rust-lightning/pull/1743/commits/08a72b8d1f1cb473dc40060da5d6108c223496e3.